src/api/models/users.js

```diff
@@ -20,14 +25,55 @@ var UsersSchema = new mongoose.Schema({
         default: Date.now
     },
     user_type: {
-        type: [{
-            type: String,
-            enum: ['student', 'recruiter']
-        }],
-        default: ['student']
+        type: String,
+        default: 'student',
+        validate: {
```

Comment:
change this validator function to be more lean. you don't need to regex test the employer_id
```
validator: function(v, cb) {
    if (v === 'student') {
        cb(this.employer_id === null);
    } else if (v === 'recruiter') {
        Employer.findById(this.employer_id).exec( function (err, employer){
            if (err || !employer) {
                cb(false, 'UserValidationError: Invalid employer_id');
            } else {
                cb(true);
            }
        })
    }
```